### PR TITLE
additional lineplot works

### PR DESCRIPTION
**Pull request by moontrip**
This PR will be used for addressing additional lineplot works. Thus branched out from `lineplot-tooltips` branch. Probably, more commits would be made in this PR in the near future.

### CODE REVIEW

**Comment by moontrip** [comment]
As the first commit, I found a way to make a line break for marker's tooltip. Here is a screenshot after resolving it.

lineplot-tooltip-linebreak

**Comment by moontrip** [comment]
@bobular in this web-components PR, I have added disabled props for `axisRangeControl` control and `Button` and `NumberAndDateRangeInputs` widgets so that they can be disabled. Also add truncation parts and axis ranges to `Lineplot` component. 

This PR works with corresponding [web-eda PR](https://github.com/VEuPathDB/web-eda/pull/1007)

File: src/plots/LinePlot.tsx [hunk 1]
```diff
@@ -151,7 +151,8 @@ const LinePlot = makePlotlyPlotComponent('LinePlot', (props: LinePlotProps) => {
               lower != null && upper != null
                 ? ` (95% CI: ${lower} - ${upper})`
                 : '';
-            return `x: ${binLabel ?? x}\ny: ${y}${CI}\n${xtra}`;
+            // use <br> instead of \n for line break
+            return `x: ${binLabel ?? x}<br>y: ${y}${CI}<br>${xtra}`;
```
Comment (bobular) [inline src/plots/LinePlot.tsx:222]: I could swear I tried `<br/>` but maybe it has to be just `<br>`

Anyway, great result!

**Review by bobular — APPROVED** [review]
All looks great 👍

**Pull request merged**